src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-wizard/rgw-multisite-wizard.component.ts

```diff
@@ -0,0 +1,258 @@
+  currentStep: WizardStepModel;
+  currentStepSub: Subscription;
+  permissions: Permissions;
+  stepTitles = ['Create Realm & Zonegroup', 'Create Zone', 'Select Cluster'];
+  stepsToSkip: { [steps: string]: boolean } = {};
+  daemons: RgwDaemon[] = [];
+  selectedCluster = '';
+  clusterDetailsArray: any;
+  isMultiClusterConfigured = false;
+  exportTokenForm: CdFormGroup;
+  realms: any;
+  loading = false;
+  pageURL: string;
+  icons = Icons;
+  rgwEndpoints: { value: any[]; options: any[]; messages: any };
+
+  constructor(
+    private wizardStepsService: WizardStepsService,
+    public activeModal: NgbActiveModal,
+    public actionLabels: ActionLabelsI18n,
+    private rgwDaemonService: RgwDaemonService,
+    private multiClusterService: MultiClusterService,
+    private rgwMultisiteService: RgwMultisiteService,
+    public notificationService: NotificationService,
+    private router: Router
+  ) {
+    this.pageURL = 'rgw/multisite';
+    this.currentStepSub = this.wizardStepsService
+      .getCurrentStep()
+      .subscribe((step: WizardStepModel) => {
+        this.currentStep = step;
+      });
+    this.currentStep.stepIndex = 1;
+    this.createForm();
+    this.rgwEndpoints = {
+      value: [],
+      options: [],
+      messages: new SelectMessages({
+        empty: $localize`There are no endpoints.`,
+        filter: $localize`Select endpoints`
+      })
+    };
+  }
+
+  ngOnInit(): void {
+    this.rgwDaemonService
+      .list()
+      .pipe(
+        switchMap((daemons) => {
+          this.daemons = daemons;
+          const daemonStatsObservables = daemons.map((daemon) =>
+            this.rgwDaemonService.get(daemon.id).pipe(
+              map((daemonStats) => ({
+                hostname: daemon.server_hostname,
+                port: daemon.port,
+                frontendConfig: daemonStats['rgw_metadata']['frontend_config#0']
+              }))
+            )
+          );
+          return forkJoin(daemonStatsObservables);
+        })
+      )
+      .subscribe((daemonStatsArray) => {
+        this.rgwEndpoints.value = daemonStatsArray.map((daemonStats) => {
+          const protocol = daemonStats.frontendConfig.includes('ssl_port') ? 'https' : 'http';
+          return `${protocol}://${daemonStats.hostname}:${daemonStats.port}`;
+        });
+        const options: SelectOption[] = this.rgwEndpoints.value.map(
+          (endpoint: string) => new SelectOption(false, endpoint, '')
+        );
+        this.rgwEndpoints.options = [...options];
+      });
+
+    this.multiClusterService.getCluster().subscribe((clusters) => {
+      this.clusterDetailsArray = Object.values(clusters['config'])
+        .flat()
+        .filter((cluster) => cluster['cluster_alias'] !== 'local-cluster');
+      this.isMultiClusterConfigured = this.clusterDetailsArray.length > 0;
+      if (!this.isMultiClusterConfigured) {
+        this.stepTitles = ['Create Realm & Zonegroup', 'Create Zone', 'Export Multi-site token'];
```

Comment:
This is because we have two cases here, one is if we have multi-cluster configured and we have added one or more clusters to it, then we show these step titles - `['Create Realm & Zonegroup', 'Create Zone', 'Select Cluster']`. In this case we allow the user to select a cluster where they want to import the multi-site configuration. The other case is when we do not have clusters added in multi-cluster and we show these titles - `['Create Realm & Zonegroup', 'Create Zone', 'Export Multi-site token']`. In this case we just show the realm token to the user and they can manually proceed to export/import the token in other clusters